linter/whitelists/de.txt

```diff
@@ -400,3 +401,5 @@ mobilfähige
 Formvalidation
 Weiterleitungszielen
 Weiterlesen
+ZIP
+Zipuploads
```

Comment:
That's not really a German nor an English word, is it? 😄 I would place this in the `default.txt` because it's a name. Then you also don't have to put it in both files. Also `ZIP` is a name as well. It can also go to `default.txt` 👍